src/Routers/ClassesRouter.js

```diff
@@ -107,6 +108,9 @@ export class ClassesRouter extends PromiseRouter {
   }
 
   handleDelete(req) {
+    if (!req.params.objectId) {
+      req.params.objectId = '*';
```

Comment:
I think instead of using `*` as a flag value, it might better to create a new handling function that just goes straight to the database adapter. This seems also seems like it might be a security hole if someone hits `DELETE /classes/:className/*`